### ISSUE TITLE
False positive on io.gitlab.arturbosh.detekt:detekt-cli due to misidentified cpe io.gitlab.gitlab

### ISSUE THREAD

**Issue by reddyalready** [dependency-check/DependencyCheck#1151]
False positive on libraries

- detekt-cli-1.0.0.RC6-4.jar - io.gitlab.arturbosch.detekt:detekt-cli:1.0.0.RC6-4 - reported as cpe:/a:gitlab:gitlab:1.0.0.rc6
- detekt-core-1.0.0.RC6-4.jar - io.gitlab.arturbosch.detekt:detekt-core:1.0.0.RC6-4 - cpe:/a:gitlab:gitlab:1.0.0.rc6
- detekt-rules-1.0.0.RC6-4.jar - io.gitlab.arturbosch.detekt:detekt-rules:1.0.0.RC6-4 - cpe:/a:gitlab:gitlab:1.0.0.rc6
- detekt-api-1.0.0.RC6-4.jar - io.gitlab.arturbosch.detekt:detekt-api:1.0.0.RC6-4 - cpe:/a:gitlab:gitlab:1.0.0.rc6

```xml
<dependency>
   <groupId>io.gitlab.arturbosch.detekt</groupId>
   <artifactId>detekt-cli</artifactId>
   <version>1.0.0.RC6-4</version>
</dependency>
```

Looks like a misidentification as the `gitlab` project due to being under the same namespace (`io.gitlab`)

**Comment by jeremylong** [comment]
Thanks for the FP report!

**Comment by reddyalready** [comment]
No problem at all!

I just realised that the fix is simply a PR to
dependencycheck-base-suppression.xml, so I will do it later today.

**Comment by reddyalready** [comment]
Actually @jeremylong, let me know if this is a bug since I feel that `cpe:/a:gitlab:gitlab` should not be picking up `io.gitlab.arturbosch`. 

If it is - I'd love to fix it as well (although may need some guidance).

Otherwise I'll submit an update to the suppression file.

**Comment by jeremylong** [comment]
@reddyalready Updating the suppression file for this is likely the best option. PRs are welcome - or I will sweep through all the FP reports in the near future and update the suppressions file as appropriate.

**Comment by jeremylong** [comment]
Thanks for the PR! This will be included in the next release.
